data_collection/gazette/spiders/pr_cianorte.py

```diff
@@ -0,0 +1,34 @@
+from dateparser import parse
+from scrapy.http import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PrCianorteSpider(BaseGazetteSpider):
+    TERRITORY_ID = "4105508"
+    name = "pr_cianorte"
+    allowed_domains = ["cianorte.pr.gov.br"]
+    start_urls = ["https://cianorte.pr.gov.br/orgao-oficial/"]
+    current_page = 1
+    base_url = "https://cianorte.pr.gov.br/orgao-oficial/pagina/"
+
+    def parse(self, response):
+        lines = response.css("tr")[1:]
+        if not lines:
+        	return
+        	
+        for row in lines:
+        ...
+                date = row.xpath("td[2]/text()").get()
+                edition_number = row.xpath("td[1]/text()").get()
+
+                yield Gazette(
+                    date=parse(date, languages=["pt"]).date(),
+                    edition_number=edition_number,
+                    file_urls=[row.css("a::attr(href)").get()],
```

Comment:
**suggestion**: It is better to be more specific which link you want to collect:
`file_urls=row.xpath("td[3]/a/@href").getall()`
(I used `get_all` so the return is already a list)